index.html

```diff
@@ -5931,7 +5931,12 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content is supplementary to some other piece of content.</p>
+				<p>In cases where an element with role <rref>note</rref> has an explicit association with another element, authors can use one of the following mechanisms to point to the note from the associated element:</p>
```

Comment:
I still worry this addition would make people think they _need_ to do this.   I most commonly come across notes in specs, and I'll use them (when i remember) in my own writing.  Notes in these cases are in line with the content they are adding supplementary info to.  e.g., `<p>...</p> <p role=note>...</p> <p>...</p>`.

If adding in this additional information, we should be very clear it is NOT for this basic usage.   Having re-read this again today after some time away, I'm having a hard time coming up with a scenario where i would advise someone to follow these additional steps.  Not saying they might not be appropriate, but each time I think of something, I talk myself out of it thinking "well that's more like a footnote pattern, using links to ensure all users can navigate to and from the supporting content."  or  "now we're overlapping with the annotation roles".

Reply:
Doesn't "In cases where..." copy suggest that the use of relationship attributes is required?

Also: Do we have a use case of a note that it is **not** associated with some other particular element? I can't think of one, although they may exist. Dissociated notes seem to be the exception, not the rule. AFAICT a note **always** has a context. A note is (arguably) a kind of annotation, although the annotation roles do not include "note" for reasons that I have not yet discovered.

Reply:
>Do we have a use case of a note that it is not associated with some other particular element?

Yes. I already mentioned it, and you referenced it in your response.  If a note is already presented in line with the content it further describes, there's no need to provide a programmatic association. It's implicit with the natural reading order of the content.  

Sure, I suppose one could consider a note a type of annotation, but I'd submit that a note is something added by the author for clarity, and is a part of the document itself.  Annotations, e.g., `comment` and `suggestion` are reactions _other_ people, but also including an author, to existing content.  They can be incorporated into a document, but are not part of that document in its initial state.

In line with my previous comment, I want to reiterate that I'm not saying there isn't a use case for associating a note with another element... but I want the spec to be clear on what that use case is.  Again, I would reference the ARIA spec as an example of where programmatically associating notes (the content blocks that start with the word "NOTE" are marked up as `role=note`) with other content isn't necessary.

Reply:
Well, the association is still there, it's just implicit and contextual, rather than explicitly and programmatically specified.

Anyway I get your point. I can see that we need to distinguish between 

- notes within rich text, which are usually contextually associated, and may have no additional need of aria relationship attributes
- notes which might appear as signposts, hints or supplementary guidance within a user interface, where the note's relationship to a particular part might need to be programmatically indicated

, where the non-operable/static parts (including any notes) would typically be 'skipped' as a user tabs around.

Reply:
resolving this thread as I have made updates to the content to address my concerns here.